gslib/tests/test_shim_util.py

```diff
@@ -1110,9 +1111,24 @@ def test_gcs_json_endpoint_translation_usees_default_version_v1(self):
       self.assertEqual(
           env_vars, {
               'CLOUDSDK_API_ENDPOINT_OVERRIDES_STORAGE':
-                  'https://foo_host:1234/storage/v1'
+                  'https://foo_host:1234/storage/v1/'
           })
 
+  def test_gcs_json_endpoint_translation_should_start_with_https(self):
+    with _mock_boto_config({'Credentials': {'gs_json_host': 'foo_host',}}):
+      flags, env_vars = self._fake_command._translate_boto_config()
+      self.assertEqual(flags, [])
+      self.assertTrue(
+          env_vars['CLOUDSDK_API_ENDPOINT_OVERRIDES_STORAGE'].startswith(
+              'https://'))
+
+  def test_gcs_json_endpoint_translation_should_end_with_trailing_slash(self):
```

Comment:
Ditto. Feels like test_gcs_json_endpoint_translation is already taking care of ensuring the endpoint ends with  a slash?

Reply:
It is the same reason as above, be explicit about the requirement that the string should end with a trailing `/`, but probably a comment is enough?

Reply:
Same here, comment should be enough.